src/appshell/qml/Preferences/internal/MidiSection.qml

```diff
@@ -29,9 +29,12 @@ BaseSection {
     title: qsTrc("appshell/preferences", "MIDI")
 
     property alias shortestNotes: shortestNotesBox.model
+    property alias roundTempo: roundTempoBox.checked
+
     property int currentShortestNote: 0
```

Comment:
Minor code style thing: the `shortestNotes` and `currentShortestNote` properties are closely related, so it would be better to keep them together. I propose to change the order like this:
```suggestion
    property alias shortestNotes: shortestNotesBox.model
    property int currentShortestNote: 0

    property alias roundTempo: roundTempoBox.checked
```